### ISSUE TITLE
The training results on the Dancetrack did not achieve the results in the paper

### ISSUE THREAD

**Issue by yyyzjing** [Xavier-Lin/TrackSSM#3]
Hello, I benefited a lot from your excellent work, thank you very much! 
The pt file you provided can achieve the effect in the paper, but when I trained by mysel, it didn't work. I used a 4090 to train dancetrack, according to the paper's settings for 120 epochs, and the other parameters were all set in accordance with the dancetrack.yaml settings, but the results submitted to codalab only had a result of HOTA: 53 or so. I would like to ask where the problem is?
Is it a problem with the training equipment?
However, when I was training dancetrack, the loss of Time_info_decoder was still written as loss = loss_l1.mean() + loss_giou.mean(). Shouldn't it only be loss = loss_l1.mean() here?
Looking forward to your reply!!!!

**Comment by yyyzjing** [comment]
****

**Comment by Xavier-Lin** [comment]
Before training TrackSSM on DanceTrack, you should confirm the following points：
0. The running environments of TrackSSM build on mamba-1.2.0 post1 , triton == 2.1.0 and NVCC(11.8)
1.  The code at 61 lines in motion_decoder.py should be enabled.
2. Not using GIoU loss
3. It is recommended to keep the settings in dancetrack.yaml unchanged, including the epoch option.

**Comment by yyyzjing** [comment]
Thank you very much for your reply!
0. My running environment configuration is the same as yours. 
1. The sigmoid function is also used in motion_decoder. 
2. However, the loss function uses loss = loss_l1.mean() + loss_giou.mean(). Here I need to modify it to loss = loss_l1.mean(). 3. But I would like to ask you about **keeping dancetrack.yaml unchanged**, I saw that the epoch setting is **800**, but the paper says it is **120**, so do you need to train for 800 epochs? 
In addition, the code seems to save every 20 epochs. **So how did you filter out the best pt file?** 

Looking forward to your reply!

**Comment by Xavier-Lin** [comment]
Firstly, train according to 800 epochs, save weights every 20 epochs, and continue training until 120 epochs are saved

**Comment by tmdwo8814** [comment]
Did you solve the problem?
